src/main/target/common_fc_pre.h

```diff
@@ -124,6 +124,7 @@
 #define VTX_CONTROL
 #define VTX_SMARTAUDIO
 #define VTX_TRAMP
+#define CAMERA_CONTROL
```

Comment:
What about `VTX_CONTROL` and all other defines then, you suggest prefixing them with `USE_` as well?